Source/diablo.cpp

```diff
@@ -1099,6 +1100,7 @@ void PressChar(int vkey)
 		return;
 	}
 	if (PauseMode == 2) {
+		if (GetAsyncKeyState(VK_MENU) & 0x8000) dx_reinit();

```

Comment:
why this? looks like just pressing alt while paused will trigger it

Reply:
Can you toggle (Alt+Enter) fullscreen if the game is paused?

Reply:
Can you toggle (Alt+Enter) fullscreen if the if the player is dead?

Reply:
It should resolve #659

Reply:
yes but isn't just the top line enough?

Reply:
Tested again. Top line only is not enough.

Reply:
```cpp
if (vkey == VK_RETURN) {
			if (GetAsyncKeyState(VK_MENU) & 0x8000) dx_reinit();
```
just put this on top o.O

Reply:
One line is for pause.
The other for the "red screen of death".

Reply:
so why not have 1 line for all? xD

Reply:
If a line is omitted, then one toggle function is missing.

Reply:
Because one line is in function "void PressKey(int vkey)"
The second on is placed in the function "void PressChar(int vkey)"

Reply:
ok my bad, thought they were in 1, still, insert new lines after ifs
and doesn't pressing alt alone toggle the fullscreen during pause now?

Reply:
Pressing alt alone doesn't toggle the fullscreen.

Reply:
yep I just checked ;)